packages/endpoints-credential-types-registrar/src/controllers/credential-types/schemas/new-credential-type.schema.json

```diff
@@ -97,6 +97,17 @@
       "type": "string",
       "enum": ["SECP256K1", "ES256", "RS256"],
       "default": "SECP256K1"
+    },
+    "primaryOrganizationClaimPaths": {
```

Comment:
Is there a migration PR in monorepo? I didn't see

Reply:
There will be